cloel.el

```diff
@@ -134,10 +134,11 @@
 
     (list start-func-name stop-func-name restart-func-name call-async-func-name call-sync-func-name send-message-func-name)))
 
-(defun cloel-register-app (app-name app-file)
+(defun cloel-register-app (app-name app-dir aliases)
```

Comment:
aliases here should be a string
example
1 alias `cloel`  -> `"cloel"`
2 aliases `cloel`, `production` -> `"cloel:production"`